src/Controls/src/Core/Handlers/Shell/ShellItemHandler.Windows.cs

```diff
@@ -432,7 +432,7 @@ void OnCurrentShellSectionPropertyChanged(object? sender, System.ComponentModel.
 			if (_mainLevelTabs == null)
 				return;
 
-			var currentItem = VirtualView.CurrentItem.CurrentItem;
+			var currentItem = VirtualView.CurrentItem?.CurrentItem;
```

Comment:
did we enable nullable here ?

Reply:
Yes, the nullable reference types are enabled here. After clearing CurrentItem, it reverts to its default value, which can indeed be null. To prevent potential exceptions, I will ensure that proper null checks are in place before accessing CurrentItem.